app/controllers/users_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
 
   def show
     @user = User.find(params[:id])
-    # @user_posts = Post.where(author_id: @user.id)
+    @post = Post.find(params[:id])
```

Comment:
- [Optional] Kindly remove this line of code as it brings an error to your code. The users show action should only display details of the user and its three most recent posts. There is no post id in the parameters to show a single post so kindly remove this line of code.
![image](https://github.com/jonas-45/my-blog-rails/assets/94551484/c7ddf9ab-32bc-451e-8e55-4115fc152111)

Reply:
Okay thanks